### PR TITLE
WIP: Add an "add" function

### PR DESCRIPTION
**Pull request by valishah**
Hi , This is my first open source pull request . Seem like everything went good.  While doing commit i have got stuck with code coverage. 

 1. add › Adds two integers
   failed with "(0 , _src.add) is not a function"

Hope to contribute more.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -30,4 +31,5 @@ export {
   isNullOrWhitespace,
   startsWith,
   removeDuplicates,
+  additon,
```
Comment (kentcdodds) [inline src/index.js]: I would prefer that this were called `add` instead of `addition`. Care to make the update?

Comment (valishah) [inline src/index.js]: Great. Sure, i would do that. Thanks a lot for quick response.

**Comment by kentcdodds** [comment]
Terrific! Thanks!

**Comment by valishah** [comment]
Thanks for your help @kentcdodds :) Can you share the url for knowing what open source projects to contribute ?

**Comment by kentcdodds** [comment]
Yeah, check [this](http://kcd.im/what-oss) out :-)

**Comment by valishah** [comment]
Thanks a lot @kentcdodds . I will be helping the people who comes here to learn what i have learned. Thanks again for all your efforts. I really appreciate that.

**Comment by kentcdodds** [comment]
Awesome!

**Pull request merged**